Services/DataX.sln

```diff
@@ -1,549 +1,559 @@
-		{01087DD2-99C7-460E-9C15-830D99F91E7A} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{8469A6A1-07B3-4FB4-8DA0-5D70BF3670BA} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{F1A9F5EC-786A-41E3-8000-4F0F0C39B5EF} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{95C73495-2D30-4009-BFF2-5A299702E205} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{0DF975F6-22FA-48F0-9E64-AEF5ED33F7F7} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{30F71B46-805D-4145-8513-5196AA88CD53} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{0DE74D1C-16D6-42DD-BEAF-DDF8D556DBFE} = {3076F883-13E8-498A-A462-B56E7EC84DD2}
-		{2E40E74F-4AAD-4BE4-9ED0-E6AC17916864} = {3076F883-13E8-498A-A462-B56E7EC84DD2}
-		{E2C549DD-9EBB-4FA0-8CF3-354B6EDF3635} = {3076F883-13E8-498A-A462-B56E7EC84DD2}
-		{1356403C-69C6-429B-AC77-38780284A658} = {98257003-9D75-4A84-8F09-6C70B5D30DA4}
-		{AC26DE00-E2C4-492F-98F4-A8BDD14F8EFD} = {98257003-9D75-4A84-8F09-6C70B5D30DA4}
-		{ADB9F794-54E1-4048-A25D-A8CE04BC9E8A} = {98257003-9D75-4A84-8F09-6C70B5D30DA4}
-		{2582E754-F7B4-48C2-A99A-067449D0AB22} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{5427FEC7-346D-4F2E-BDBB-F23F4846A988} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{EE98CC90-36F9-40DB-91B9-1229A2111EDC} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{E1774F38-624E-4D0D-B2E0-6F9D9FD1F553} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{B110FCAA-F211-4307-AC08-F2BC7B3D780E} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{63943442-05D9-4C96-88D8-440CE768095C} = {4B51D86C-636D-442A-81BC-A209D51B76E2}
-		{7B9A41AC-79D3-4A1C-892A-97BF582872AE} = {1B0E5AB0-0E00-4587-91B5-59874F9FAAA6}
-		{0664265C-D454-4F19-9CD0-1A211EDBE64F} = {1B0E5AB0-0E00-4587-91B5-59874F9FAAA6}
-		{94904438-0110-476D-9F7D-674ACFC5894D} = {1B0E5AB0-0E00-4587-91B5-59874F9FAAA6}
-		{ABB75BEF-3017-450A-BB85-F7EAD38E5EBD} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{DECD054B-B309-4831-980E-762DA443FB19} = {E2D64F0A-15EE-400F-967A-608F3794A5D0}
-		{4F8CCFBD-871B-41D3-AB86-1257E91B9A41} = {E2D64F0A-15EE-400F-967A-608F3794A5D0}
-		{C5631493-E37C-46C6-B63E-42A97A9F722F} = {E2D64F0A-15EE-400F-967A-608F3794A5D0}
-		{78CD8E8F-EA26-446A-922C-F6A88CE6551A} = {1B0E5AB0-0E00-4587-91B5-59874F9FAAA6}
-		{8F672B7A-4515-476E-9643-2F4BEF68306B} = {98257003-9D75-4A84-8F09-6C70B5D30DA4}
-		{702A1D3B-63AF-4519-8334-600E477D9CBC} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{13288F78-76B4-4B9F-B57B-A7415FEEF42F} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{20E1DF46-301A-4F74-8E76-D29192E368E5} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{C1CAC657-0322-4275-AE64-EC83FDA8C820} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{9E338FCC-793F-4048-9672-048270C856C4} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-		{D5D3DD2D-760A-4818-A74F-C0061719841C} = {6FF7998A-7952-444B-AD85-E3A9EB8683C1}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {FB461262-8931-453E-8214-506BF892338E}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15

```

Comment:
Do we need this change? if so, why?

Reply:
This was always 15. We can make a pass for all of the solutions in our repo and move them all to 16. For now, for consistency, wil leave this as 15.